config_builder.go

```diff
@@ -719,6 +719,41 @@ func (d *DefaultWalletImpl) BuildChainControl(
 				partialChainControl.ChainNotifier,
 			),
 			RebroadcastInterval: pushtx.DefaultRebroadcastInterval,
+			// In case the backend is different from neutrino we
+			// make sure that broadcast backend errors are mapped
+			// to the neutrino broadcastErr.
+			MapCustomBroadcastError: func(err error) error {
```

Comment:
nit: could make a `BroadcastErrorMapper` function and assign it to `MapCustomBroadcastError` so we won't grow the already too long method `BuildChainControl`😂

Reply:
I did use `broadcastErrorMapper ` so unexported, I don't think we need to export it or?